src/Symfony/Component/DependencyInjection/Compiler/AutowirePass.php

```diff
@@ -459,6 +424,9 @@ private function addServiceToAmbiguousType($id, $type)
         $this->ambiguousServiceTypes[$type][] = $id;
     }
 
+    /**
+     * @deprecated since version 3.3, to be removed in 4.0.
```

Comment:
It's just a friendly reminder for the one who will cleanup the code preparing 4.0